From 13034f443b4c6763d59e220cf4e9936c81718e49 Mon Sep 17 00:00:00 2001
From: Jason Zeng <jason.zeng@intel.com>
Date: Mon, 15 Aug 2022 17:18:34 +0800
Subject: [PATCH 0462/1646] [Backport] Enable Intel AMX for default config

Intel inclusion
category: feature
bugzilla: https://gitee.com/openeuler/intel-kernel/issues/I5MJBR

Reference: https://gitee.com/openeuler/kernel/commit/5e8552d8c2a10b37e07bbd81695b2151b7a55989

--------------------------------------------

Enable necessary kernel configs for Intel AMX in openeuler_defconfig.

Reviewed-by: Lin Wang <lin.x.wang@intel.com>
Signed-off-by: Jason Zeng <jason.zeng@intel.com>
Signed-off-by: Aichun Shi <aichun.shi@intel.com>
Signed-off-by: Jialin Zhang <zhangjialin11@huawei.com>
---
 arch/x86/configs/openeuler_defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/x86/configs/openeuler_defconfig b/arch/x86/configs/openeuler_defconfig
index 80ac17cf1fad..72039dee1c5e 100644
--- a/arch/x86/configs/openeuler_defconfig
+++ b/arch/x86/configs/openeuler_defconfig
@@ -484,6 +484,7 @@ CONFIG_LEGACY_VSYSCALL_EMULATE=y
 # CONFIG_LEGACY_VSYSCALL_NONE is not set
 # CONFIG_CMDLINE_BOOL is not set
 CONFIG_MODIFY_LDT_SYSCALL=y
+# CONFIG_STRICT_SIGALTSTACK_SIZE is not set
 CONFIG_HAVE_LIVEPATCH_FTRACE=y
 CONFIG_HAVE_LIVEPATCH_WO_FTRACE=y
 
@@ -851,6 +852,7 @@ CONFIG_HAVE_STATIC_CALL=y
 CONFIG_HAVE_STATIC_CALL_INLINE=y
 CONFIG_HAVE_PREEMPT_DYNAMIC=y
 CONFIG_ARCH_WANT_LD_ORPHAN_WARN=y
+CONFIG_DYNAMIC_SIGFRAME=y
 
 #
 # GCOV-based kernel profiling
-- 
2.17.1

